src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsHopObject.java

```diff
@@ -33,13 +34,21 @@ public AdsHopObject(WebDriver driver, String page) {
     super(driver, page);
   }
 
-  public AdsHopObject verifyClassHidden(String slotName, String src) {
-    WebElement testedDiv = getTestedDiv(slotName, src);
-    Assertion.assertEquals(testedDiv.getAttribute("class").trim(), "hidden");
-    return this;
+  public void verifyClassHidden(final String slotName, final String src) {
+    new WebDriverWait(driver, AD_SUCCESS_TIMEOUT_SEC).until(new ExpectedCondition<Boolean>() {
+      @Override
+      public Boolean apply(WebDriver driver) {
+        return getTestedDiv(slotName, src).getAttribute("class").trim().equals("hidden");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the "hidden" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1132)